### DISCUSSION TITLE
CSV import job is not executed

### DISCUSSION THREAD

**Discussion by AHMED-GAMAL-AG** [filamentphp/filament#10624] [Help]
I am working on a project migrating from an ASP.NET to a laravel one while keeping the SQL Server database and I am using the filament dashboard, I can't use the new import from CSV feature I tried to use it on my User model Note: that I have to change the table name and columns to match the SQL Server

![image](https://github.com/filamentphp/filament/assets/76778937/66350772-ca7e-4b1a-8fad-87e91102963b)

the incrementing of the Ids is saved in a SQL Server Sequence so I have to manually Retrieve the next value from the Sequence when creating a new record like so

![image](https://github.com/filamentphp/filament/assets/76778937/112aa3d2-0a81-4350-9ec2-267afc74a873)

Now when using the import feature I get a message that the Import has started

![Screenshot (1299)](https://github.com/filamentphp/filament/assets/76778937/2b373751-32d1-4482-84dc-d09f6c7d3c13)

but the job is never executed and I have my queue_driver on SYNC, so it should run directly

I also tried to manually set the Id in the Importer but that also didn't work i got Import started with no errors 

![image](https://github.com/filamentphp/filament/assets/76778937/7536f4a5-b575-4dbd-8e53-7cdba12c8013)

Note that I changed the default user model 

![image](https://github.com/filamentphp/filament/assets/76778937/2d6d4ead-274c-4d70-a39e-ab0103a3ff29)
![image](https://github.com/filamentphp/filament/assets/76778937/06f7913f-3167-471e-b6e1-dcaa42d62e38)

**Comment by mayzin00** [comment]
I'm also facing this issue. Could you please share me knowledge of how you handled it?

**Comment by AHMED-GAMAL-AG** [comment]
after deep debugging, I couldn't find a solution. I didn't have time to debug more I had to import the data manually using this package

[https://github.com/eighty9nine/filament-excel-import](url)

you could use any other package but that is the one that worked with me

**Comment by sandeep0108** [comment]
I am getting the same issue on server, but working in local system
And followed the same https://filamentphp.com/docs/3.x/actions/prebuilt-actions/import#customizing-the-import-queue-and-connection

**Reply by pvdptje** [reply]
queue_connection=database  instead of queue_connection=sync
